stdlib/src/builtin/tuple.mojo

```diff
@@ -178,3 +180,36 @@ struct Tuple[*element_types: Movable](Sized, Movable):
             The tuple element at the requested index.
         """
         return rebind[T](self[i])
+
+    @always_inline("nodebug")
+    fn __contains__[
+        RHS_T: ComparableCollectionElement
```

Comment:
Please use PascalCase for type parameters.

Reply:
Yes, sorry about that, i do remember it all the time i see parameters now so thanks  :+1: 

Have got a better implementation, safer with references, please do not merge this one yet if it was the plan;

I'll do a force-push later-on, still don't know if it will work with the rebased!